docs/reset-mfa.md

```diff
@@ -1,4 +1,17 @@
 # Reset MFA
 
-- [Reset security verification for WOG account](reset-security-verification-for-wog-account)
-- [Reset security verification for TechPass](reset-techpass-mfa-for-new-device)
+TechPass users need to reset MFA when they do any of the following:
+
+- Change their mobile phone.
+- Delete Authenticator app from their mobile phone.
+- Get a new TechPass account.
```

Comment:
it's not reset mfa for new accounts right? it's setup actually. so don't think the last option is applicable

Reply:
If a vendor who has changed organisation (say from cognizant to techmahindra) , he will get a new techpass account and in that case should he follow reset TechPass (because he may still have the old profile on his authenticator or is that out od scope for reset mfa) That was my intention while adding the third use case

similarly, what if a officer changes agencies and gets a new WOG account? Should that also be out od scopr of this doc?

Reply:
noted. thanks.